### ISSUE TITLE
Plugin should check the WP Database when user isnt found in Auth0 Database

### ISSUE THREAD

**Issue by lots0logs** [auth0/wp-auth0#107]
I have it configured to do SSO with our forum. I'm seeing an issue when trying to log in with a user that is in the WP database but not in the forum database. Here is the log entry:

**Occurred:** a few seconds ago
**Type:** Failed Login (wrong password)
**Description:** id is required
**Connection:** Username-Password-Authentication
**Application:** *removed* (*removed*)
**User:** *removed*
```json
{
  "date": "2015-12-12T10:27:11.617Z",
  "type": "fp",
  "description": "id is required",
  "connection": "Username-Password-Authentication",
  "client_id": "removed",
  "client_name": "removed",
  "ip": "removed",
  "user_agent": "unknown",
  "user_id": "auth0|unknown",
  "user_name": "removed",
  "strategy": "auth0",
  "strategy_type": "database"
}
```
```json
{
  "error": {
    "message": "id is required"
  }
}
```

**Comment by glena** [comment]
Hi, where do you see this error?

By default, the plugin creates the user if it does not exists in the WP database.

**Comment by lots0logs** [comment]
The user **does exist** in the WP Database though. After I created this issue I went back and re reviewed the readme for this plugin and I realized that the fact that the plugin does not check the WP User Database is intentional. Though I think you guys should reconsider it and here's why: 

When you have an existing database of users prior to implementing Auth0, you'll probably want to migrate those users. With custom Database connections this is simple enough from the Auth0 Dashboard. However, since you have an actual plugin that integrates WordPress,why should the user need to create yet another custom database connection in order to migrate the users? IMO, the plugin should check the WP Database automatically without additional setup. 

Though I realize that in order to keep things consistent on the Auth0 side the WordPress plugin would probably need to create a new custom Database connection for itself anyway. Obviously, from the user POV its better for the plugin to handle all of this. However, at the very least, the documentation needs to be improved to explain what needs to be done to make it work. The current documentation makes it sound like it will take care of everything when in fact it does not.
